src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -149,25 +151,32 @@ const BlastSettings = ({ config }: Props) => {
           <h1 className={styles.title}>Blast</h1>
           <div className={styles.runJob}>Run a job</div>
           <div>
-            {buildSelect({
-              ...(config.parameters['database'] as BlastSelectSetting),
-              selectedOption: blastParameters.database as string,
-              onChange: onDatabaseChange
-            })}
+            <BlastSelect
+              options={config.parameters.database.options as Option[]}
+              label={config.parameters.database.label}
+              description={config.parameters.database.description as string}
+              selectedOption={blastParameters.database as string}
+              onChange={onDatabaseChange}
+              showTooltip={false}
+            />
           </div>
           <div>
-            {buildSelect({
-              ...(availableBlastPrograms as BlastSelectSetting),
-              selectedOption: blastProgram,
-              onChange: onBlastProgramChange
-            })}
+            <BlastSelect
+              options={availableBlastPrograms.options}
+              label={availableBlastPrograms.label}
+              description={availableBlastPrograms.description as string}
+              selectedOption={blastProgram}
+              onChange={onBlastProgramChange}
+              showTooltip={false}
+            />
           </div>
           <div>
-            {buildSelect({
-              ...(getPresetsList(config) as BlastSelectSetting),
-              selectedOption: searchSensitivity,
-              onChange: onSearchSensitivityChange
-            })}
+            <BlastSelect
+              options={getPresetsList(config).options}
+              label={getPresetsList(config).label}
+              selectedOption={searchSensitivity}
+              onChange={onSearchSensitivityChange}
```

Comment:
showTooltip = false is missing here?

Reply:
yes, but at the same time there is no description, which means it will not be displayed. I can add that just for readability